src/IBM.Watson.TextToSpeech.v1/Test/Integration/TextToSpeechServiceIntegrationTests.cs

```diff
@@ -264,7 +265,282 @@ public void Words_Success()
+                FileStream fs = File.OpenRead(wavFile);
+                fs.CopyTo(ms);
+                var speakerModel = service.CreateSpeakerModel(
+                    speakerName: "speakerNameDotNet",
+                    audio: ms
+                    );
+                speakerId = speakerModel.Result.SpeakerId;
+
+                Assert.IsNotNull(speakerModel.Result.SpeakerId);
+            }
+            catch (Exception ex)
+            {
+                Console.WriteLine(ex.StackTrace);
+            }
+            finally
+            {
+                service.DeleteSpeakerModel(
+                    speakerId: speakerId
+                    );
+            }
+        }
+
+        [TestMethod]
+        public void testListSpeakerModel()
+        {
+            service.WithHeader("X-Watson-Test", "1");
+            string speakerId = "";
+            try
+            {
+                MemoryStream ms = new MemoryStream();
+                FileStream fs = File.OpenRead(wavFile);
+
+                fs.CopyTo(ms);
+
+                var speakerModel = service.CreateSpeakerModel(
+                    speakerName: "speakerNameDotNet",
+                    audio: ms
+                    );
+
+                speakerId = speakerModel.Result.SpeakerId;
+
+                Assert.IsNotNull(speakerModel.Result.SpeakerId);
+
+                var speakers = service.ListSpeakerModels();
+
+                Assert.IsNotNull(speakers.Result._Speakers);
+                Assert.IsTrue(speakers.Result._Speakers.Count > 0);
+            }
+            catch (Exception ex)
+            {
+                Console.WriteLine(ex.StackTrace);
+            }
+            finally
+            {
+                service.DeleteSpeakerModel(
+                    speakerId: speakerId
+                    );
+            }
+        }
+
+        [TestMethod]
+        public void testGetSpeakerModel()
+        {
+            service.WithHeader("X-Watson-Test", "1");
+            string speakerId = "";
+            try
+            {
+                MemoryStream ms = new MemoryStream();
+                FileStream fs = File.OpenRead(wavFile);
+
+                fs.CopyTo(ms);
+
+                var speakerModel = service.CreateSpeakerModel(
+                    speakerName: "speakerNameDotNet",
+                    audio: ms
+                    );
+
+                speakerId = speakerModel.Result.SpeakerId;
+
+                Assert.IsNotNull(speakerModel.Result.SpeakerId);
+
+                var response = service.GetSpeakerModel(
+                    speakerId: speakerId
+                    );
+
+                Assert.IsNotNull(response.Result.Customizations);
+            }
+            catch (Exception ex)
+            {
+                Console.WriteLine(ex.StackTrace);
+            }
+            finally
+            {
+                service.DeleteSpeakerModel(
+                    speakerId: speakerId
+                    );
+            }
+        }
```

Comment:
I believe you are missing a `TestDeleteSpeakerModel`

Reply:
I didn't create one since I'm pretty much using it on multiple tests heh

Reply:
I would still make one to be consistent as your CustomPrompt tests do have a Delete test